CMakeLists.txt

```diff
@@ -79,6 +79,12 @@ option (SEQAN_ARCH_AVX2 "Build AVX optimized binaries." OFF)
 option (SEQAN_ARCH_NATIVE "Build natively optimized binaries." OFF)
 
 # ===========================================================================
+# Version Check Variable.
+# ===========================================================================
+
+option (SEQAN_VERSION_CHECK "SeqAn version check." ON)
```

Comment:
Is this the default value for the SEQAN_VERSION_CHECK variable? If so I should change it to OFF since in 2.3 we do now want it to be enabled by default yet, right?